src/Libraries/RevitNodesUI/Selection.cs

```diff
@@ -576,7 +600,7 @@ protected override IEnumerable<Reference> ExtractSelectionResults(Reference sele
      IsVisibleInDynamoLibrary(false)]
     public class DSAnalysisResultSelection : ElementSelection<Element>
     {
-        private const string message = "Select an analysis result.";
+        private static string message = Resources.SelectAnalysisResultsMessage;
```

Comment:
Opportunistic localization additions.  We have added the _message_ string in some cases but not all.